lib/elixir/pages/getting-started/structs.md

```diff
@@ -20,7 +19,7 @@ Structs are extensions built on top of maps that provide compile-time checks and
 
 ## Defining structs
 
-To define a struct, the `defstruct/1` construct is used:
+To define a struct, the [`defstruct/1`](https://hexdocs.pm/elixir/Kernel.html#defstruct/1) construct is used:
```

Comment:
I would say that this is bug in ExDoc if it is not autodiscovered. But IMHO that should not use absolute link to HexDocs. I think that there is some other way to cross-link, but someone more familiar with ExDoc need to point that out.